code/modules/mob/living/carbon/species.dm

```diff
@@ -1281,6 +1281,34 @@ var/list/has_died_as_golem = list()
 				to_chat(user, "<span class='notice'>You place \the [O] into \the [src].</span>")
 				qdel(O)
 
+/datum/species/insectoid
+	name = "Insectoid"
+	icobase = 'icons/mob/human_races/r_insect.dmi'
+	deform = 'icons/mob/human_races/r_def_insect.dmi'
+	known_languages = list(LANGUAGE_INSECT)
+	primitive = /mob/living/carbon/monkey/roach
+
+	flags = IS_WHITELISTED
+	anatomy_flags = HAS_LIPS | HAS_SWEAT_GLANDS
+
+	default_mutations=list(RAD_IMMUNE)
+	brute_mod = 1.3
+	burn_mod = 1.3
+	tox_mod = 0.5
+
+	blood_color = "#e6e600"
+	flesh_color = "#9c7f25"
+
+	has_mutant_race = 0
+
+	species_intro = "You are an Insectoid.<br>\
+					Your body is utterly immune to the perils of radiation, and you are able to better defend against toxic chemicals <br>\
+					However, your rigid body is somewhat more fragile than that of more soft-bodied species. Resilient though you may be, a good smack may put you out of comission."
+
+/datum/species/skrell/gib(mob/living/carbon/human/H) //placeholder til I get this figured
```

Comment:
Might be an accidental copypaste botch, did you mean `/datum/species/insectoid/gib`?

Reply:
I ask because I did something similar and it was the reason Lamprey kept crashing for months on end

Reply:
no, the gib animation's a WIP, I'm just stumped as to actual causes